packages/backend/src/generated/swagger.json

```diff
@@ -91,14 +91,20 @@
                     },
                     {
                         "properties": {
+                            "tableGroupLabel": {
+                                "type": "string"
+                            },
+                            "tableName": {
+                                "type": "string"
+                            },
                             "basicType": {
                                 "type": "string"
                             },
                             "type": {
                                 "$ref": "#/components/schemas/CatalogType.Field"
                             }
                         },
-                        "required": ["basicType", "type"],
```

Comment:
did you remove the basicType ? this was used in the metadata panel

Reply:
I didn't intend to. Maybe a bad merge?

Reply:
Irakli is adding it back on his PR , if you didn't change any types in this PR I'd suggest to revert these files